pkg/collector/networkoutbound_collector.go

```diff
@@ -57,12 +50,6 @@ func (collector *NetworkOutboundCollector) Collect() error {
 			URL:  "kubernetes.default.svc.cluster.local:443",
 		},
 	)
-	outboundTypes = append(outboundTypes,
```

Comment:
🐛 Can we not make this removal part of this PR please, we don't know why it existed, @rzhang628 , @qpetraroia or @palma21 - If anyone of you have any insight for this please? removing this means taking small functionality out of periscope and since its in `Azure` org I would prefer wider eyes on this please. Thanks

Reply:
what should we set instead?